test/basic-querying.test.js

```diff
@@ -11,6 +11,7 @@ const async = require('async');
 const bdd = require('./helpers/bdd-if');
 const should = require('./init.js');
 const uid = require('./helpers/uid-generator');
+const parentRefHelper = require('./helpers/parentRef-helper');
```

Comment:
We use kebab-case for file names.

```suggestion
const parentRefHelper = require('./helpers/parent-ref-helper');
```

Considering the actual content of this file, I think the following would be a more descriptive name: `helpers/setup-parent-ref`.

Reply:
used kebab-case for filename